packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 46.16.0 - September 26, 2019
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Retrieve Multiple Components by Metadata Type ([PR #1667](https://github.com/forcedotcom/salesforcedx-vscode/pull/1667))
```

Comment:
```suggestion
- Add retrieve all instances of a single metadata type for the Org Browser ([PR #1667](https://github.com/forcedotcom/salesforcedx-vscode/pull/1667))
```

Reply:
This is how we've called it in the Iteration Plan https://github.com/forcedotcom/salesforcedx-vscode/issues/1613

Reply:
Could rephrase as:
Update Org Browser to retrieve all instances of a metadata type